owncloudApp/src/main/res/values/strings.xml

```diff
@@ -369,6 +369,8 @@
 
     <string name="set_available_offline">Set as available offline</string>
     <string name="unset_available_offline">Unset as available offline</string>
+    <string name="confirmation_set_available_offline">The file was set as available offline correctly</string>
+    <string name="confirmation_unset_available_offline">The file was set as unavailable offline correctly</string>
```

Comment:
The name of the state is `available offline`, so let's not say that a file is set as `unavailable offline`, better say that a file was unset as `available offline`, just as the button in the menu indicates

Reply:
I've changed the string (`The file was unset as available offline correctly`)